client/app/locales/locale-de.json

```diff
@@ -23,7 +23,8 @@
     "REDIRECT": "Umleitung zu deiner Gruppe..."
   },
   "TOPBAR": {
-    "LOGOUT": "Ausloggen"
+    "LOGOUT": "Ausloggen",
+    "GREETING": "Moin, {{username}}!"
```

Comment:
We need to enter this translation in transifex later, otherwise it gets lost on update.